src/App.jsx

```diff
@@ -1,46 +1,13 @@
-import React, { useState } from 'react';
-
-import Page from './Page';
+import React from 'react';
+import InputContainer from './InputContainer';
+import ListContainer from './ListContainer';
```

Comment:
모듈과 컴포넌트들 사이에 공백을 추가하는건 어떤가요? 

```suggestion
import React from 'react';

import InputContainer from './InputContainer';
import ListContainer from './ListContainer';
```